src/backend/distributed/deparser/ruleutils_13.c

```diff
@@ -7306,7 +7306,8 @@ get_from_clause_item(Node *jtnode, Query *query, deparse_context *context)
 			/* Reconstruct the columndef list, which is also the aliases */
 			get_from_clause_coldeflist(rtfunc1, colinfo, context);
 		}
-		else if (GetRangeTblKind(rte) != CITUS_RTE_SHARD)
+		else if (GetRangeTblKind(rte) != CITUS_RTE_SHARD ||
+			     (rte->alias != NULL && rte->alias->colnames != NIL))
```

Comment:
```suggestion
				 (rte->alias != NULL && rte->alias->colnames != NIL))
```

I wonder why `citus_indent` did not catch this.

Similar formatting can be applied to other `ruleutils_1X.c` files.